src/modelmanager.hpp

```diff
@@ -367,6 +405,35 @@ class ModelManager {
      * @brief Updates OVMS configuration with cached configuration file. Will check for newly added model versions
      */
     Status updateConfigurationWithoutConfigFile();
+
+    /**
+     * @brief Cleaner thread procedure to cleanup resources that are not used
+     */
+    void cleanupResources();
```

Comment:
Protected/private?

Reply:
Cannot make it private since FunctorResourcesCleaner uses it.